nh_eobs/views/observation_report_template.xml

```diff
@@ -5,7 +5,7 @@
             <div class="row footer col-xs-12">
                 <div class="col-xs-9">
                         <p>
-                            <img src="/nh_eobs/static/src/img/open_eobs_logo.png" alt="Open-eObs" style="height: 16px;"/>
```

Comment:
Closed EOBS-323 and added a comment explaining the work was done under this issue.